src/utils/functions.rs

```diff
@@ -1,7 +1,16 @@
-#[cfg(windows)]
 use super::sys::winapi::ansi::set_virtual_terminal_processing;
 
-#[cfg(windows)]
+const UNSUPPORTED_WINDOWS_CSI_SEQUENCES: [&str; 2] = [
```

Comment:
I don't have a solution for this right now, but I'd say that this should be handled on the `impl Command for X` level. Not outside as an array. Something like:

```rust
trait Command {
    #[cfg(windows)]
    fn is_ansi_code_supported(&self) -> bool { true }
}
```

* By default, it says `true` for all ANSI codes
* You can overwrite it in the `impl Command for X` block and return `false`
* Handle it in the `queue!`, `execute!` macros via `is_ansi_code_supported()`

Reply:
I tried different implementations. This one is a nice idea as well. I will experiment with this.

Reply:
I didn't do it this way, because this is just the case for a few commands. However, that default implementation is perfect for this. We can even tell here if the system supports escape codes at all. Like for windows 7 we always want to return false.

Reply:
https://github.com/crossterm-rs/crossterm/pull/326/commits/1a6a36562b006fe2a2c7a71209a473689e68ef61